File: my-journey/dsa/cp/week-01/cpp/F-digits-summation.cpp
// Problem: F. Digits Summation
// Pattern:
// Link: https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/F
// Difficulty:
// Time:  O()
// Space: O()

#include <iostream>
using namespace std;

void solve() {
    long long a, b;
    cin >> a >> b;
    cout << (a % 10) + (b % 10) << "\n";
}

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);

    solve();
    return 0;
}

File: my-journey/dsa/week-01/python/F-digits-summation.py
# Problem: F. Digits Summation
# Pattern:
# Link: https://codeforces.com/group/MWSDmqGsZm/contest/219158/problem/F
# Difficulty:
# Time:  O()
# Space: O()

import sys
input = sys.stdin.readline


def solve():
    a, b = map(int, input().split())
    # your logic here
    last_digit_a = a%10
    last_digit_b = b%10
    result = last_digit_a + last_digit_b
    print(result)


if __name__ == "__main__":
    solve()
